buildah-build/Dockerfile

```
@@ -0,0 +1,15 @@
+# Referenced from From https://github.com/shipwrigh-io/sample-go
+
+FROM registry.access.redhat.com/ubi8/go-toolset:1.18 AS build
```

We should use a newer image - I don't think this will succeed. Also recommend using UBI9 instead of 8

```suggestion
FROM registry.access.redhat.com/ubi9/go-toolset:1.21 AS build
```